lms/services/upsert.py

```diff
@@ -45,7 +45,7 @@ def bulk_upsert(
         # the same return type in all branches.
         return db.query(model_class).filter(False)
 
-    index_elements_columns = [column(c) for c in index_elements]
+    index_elements_columns = [getattr(model_class, c) for c in index_elements]
```

Comment:
This definitely hasn't changed/broken anything? You tested that code that uses `bulk_upsert()` still works? The SQLAlchemy cleverness in this function worries me, it'd be nice to remove the cleverness honestly